### ISSUE TITLE
remove mode 1 from GMET?  

### ISSUE THREAD

**Issue by andywood** [NCAR/GMET#29]
mode 1 (downscaling grids) has not been maintained since GARD was written.  It could be removed from the code.

**Comment by anewman89** [comment]
Agree, this should have been done several years ago, but no time like the present to finally do it.
